hack/build/config.sh

```diff
@@ -19,15 +19,13 @@ APISERVER="cdi-apiserver"
 UPLOADPROXY="cdi-uploadproxy"
 UPLOADSERVER="cdi-uploadserver"
 OPERATOR="cdi-operator"
-CDI_OLM_CATALOG="cdi-olm-catalog"
 FUNC_TEST_INIT="cdi-func-test-file-host-init"
 FUNC_TEST_HTTP="cdi-func-test-file-host-http"
 FUNC_TEST_REGISTRY="cdi-func-test-registry"
 FUNC_TEST_REGISTRY_POPULATE="cdi-func-test-registry-populate"
 FUNC_TEST_REGISTRY_INIT="cdi-func-test-registry-init"
-CSV_GENERATOR="csv-generator"
```

Comment:
should this stay?

Reply:
Same answer, if we don't care about upstream CSV generator, then we don't need to compile and install it in the container.